pkg/config/defaults.go

```diff
@@ -15,17 +15,19 @@ const (
 	LogFormatJSON = "json"
 
 	// Non configurable stuff
-	defaultIdentityURLEu          = "https://identity-api.eu.newrelic.com"
-	defaultIdentityStagingURLEu   = "https://staging-identity-api.eu.newrelic.com"
-	defaultCmdChannelURLEu        = "https://infrastructure-command-api.eu.newrelic.com"
-	defaultCmdChannelStagingURLEu = "https://staging-infrastructure-command-api.eu.newrelic.com"
-	defaultCmdChannelURL          = "https://infrastructure-command-api.newrelic.com"
-	defaultCmdChannelStagingURL   = "https://staging-infrastructure-command-api.newrelic.com"
-	defaultIdentityURL            = "https://identity-api.newrelic.com"
-	defaultIdentityStagingURL     = "https://staging-identity-api.newrelic.com"
-	baseCollectorURL              = "https://%sinfra-api.%snewrelic.com"
-	baseDimensionalMetricURL      = "https://%smetric-api.%snewrelic.com"
-	defaultSecureFederalURL       = "https://gov-infra-api.newrelic.com"
+	defaultIdentityURLEu             = "https://identity-api.eu.newrelic.com"
+	defaultIdentityStagingURLEu      = "https://staging-identity-api.eu.newrelic.com"
+	defaultCmdChannelURLEu           = "https://infrastructure-command-api.eu.newrelic.com"
+	defaultCmdChannelStagingURLEu    = "https://staging-infrastructure-command-api.eu.newrelic.com"
+	defaultCmdChannelURL             = "https://infrastructure-command-api.newrelic.com"
+	defaultCmdChannelStagingURL      = "https://staging-infrastructure-command-api.newrelic.com"
+	defaultIdentityURL               = "https://identity-api.newrelic.com"
+	defaultIdentityStagingURL        = "https://staging-identity-api.newrelic.com"
+	baseCollectorURL                 = "https://%sinfra-api.%snewrelic.com"
+	baseDimensionalMetricURL         = "https://%smetric-api.%snewrelic.com"
+	defaultSecureFederalURL          = "https://gov-infra-api.newrelic.com"
```

Comment:
what's this one for?

Reply:
This one previously existed, and I believe is your collector URL, yes?

Reply:
I don't see it being used though

Reply:
https://github.com/newrelic/infrastructure-agent/blob/master/pkg/config/config.go#L1451-L1457

and

https://github.com/newrelic/infrastructure-agent/blob/master/pkg/config/config.go#L1505-L1515

Reply:
No problem :) Thanks for the review!